code/_onclick/hud/screen_objects.dm

```diff
@@ -126,13 +126,16 @@
 		if(!C.incapacitated())
 			if(C.internal)
 				C.internal = null
-				C << "<span class='notice'>No longer running on internals.</span>"
+				C << "<span class='notice'>You are no longer running on internals.</span>"
 				icon_state = "internal0"
 			else
 				if(!istype(C.wear_mask, /obj/item/clothing/mask))
-					C << "<span class='notice'>You are not wearing a mask.</span>"
+					C << "<span class='warning'>You are not wearing a mask!</span>"
 					return 1
 				else
+					var/obj/item/clothing/mask/M = C.wear_mask

```

Comment:
You could fix  7276 and 8771 by adding:
`if( !(wear_mask.flags & MASKINTERNALS) )`
`  C << "<span class='warning'>You need an internals mask blahblahblah !</span>"`
`  return`